src/components/sh-link.ts

```diff
@@ -101,13 +101,24 @@ class Link extends LitElement {
     };
   }
 
+  onMouseEvent(text: string) {
```

Comment:
I would rename this to something else because it is not _only_ happening on mouse events. Something like `dispatchFocusChange` or `dispatchTitleChange`.